cfgov/jobmanager/tests/test_models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import IntegrityError
+from django.test import TestCase
+from model_mommy import mommy
+
+from jobmanager.models import Job
+
+
+class JobTestCase(TestCase):
+    def test_slug_uniqueness(self):
+        mommy.make(Job, slug='abcde')
+        with self.assertRaises(IntegrityError):
+            mommy.make(Job, slug='abcde')
+
+    def test_slug_case_sensitive_uniqueness(self):
+        mommy.make(Job, slug='abcde')
+        mommy.make(Job, slug='ABCDE')
```

Comment:
perhaps then the test should test that an error is raised if uniqueness gets violated, i.e. assert_integrity_error_for_uniqueness_violation, and do an assertRaises(IntegrityError).

We want to ensure that this error is being raised.  That's the test, IMO.